.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+on:
+  push:
+    branches:
+    - "master"
+  pull_request:
+    branches:
+    - "master"
+
+name: Run Tox tests
+
+jobs:
+  tox_test:
+    name: Tox test
+    steps:
+    - uses: actions/checkout@v3
+    - name: Run Tox tests
+      id: test
+      uses: fedora-python/tox-github-action@main
+      with:
+        tox_env: ${{ matrix.tox_env }}
+        dnf_install: "python3-test python3.*-test pypy3-test"
```

Comment:
```suggestion
        dnf_install: "python3*-test pypy3*-test"
```

Reply:
Are you sure?
```
# dnf install python3-test python3.*-test pypy3.*-test
Installing:
 pypy3.10-test                 x86_64     7.3.13-1.3.10.fc38        updates              11 M
 pypy3.9-test                  x86_64     7.3.13-1.3.9.fc38         updates             8.3 M
 python3-test                  x86_64     3.11.8-1.fc38             updates-testing      15 M
 python3.10-test               x86_64     3.10.13-2.fc38            updates              10 M
 python3.12-test               x86_64     3.12.2-1.fc38             updates-testing      16 M
 python3.13-test               x86_64     3.13.0~a3-1.fc38          updates              16 M
Upgrading:
 python3                       x86_64     3.11.8-1.fc38             updates-testing      28 k
 python3-libs                  x86_64     3.11.8-1.fc38             updates-testing     9.6 M
…
Install  42 Packages
Upgrade   2 Packages
```
vs
```
# dnf install python3*-test pypy3*-test
Installing:
 pypy3.10-test                 x86_64   7.3.13-1.3.10.fc38            updates            11 M
 pypy3.9-test                  x86_64   7.3.13-1.3.9.fc38             updates           8.3 M
 python3-gensim-test           noarch   0.10.0-33.fc38                fedora            1.7 M
 python3-jupyter-kernel-test   noarch   0.6.0-1.fc38                  updates            31 k
 python3-samba-test            x86_64   2:4.18.10-1.fc38              updates           1.4 M
 python3-subunit-test          noarch   1.4.2-2.fc38                  fedora            104 k
 python3-test                  x86_64   3.11.8-1.fc38                 updates-testing    15 M
 python3.10-test               x86_64   3.10.13-2.fc38                updates            10 M
 python3.12-test               x86_64   3.12.2-1.fc38                 updates-testing    16 M
 python3.13-test               x86_64   3.13.0~a3-1.fc38              updates            16 M
Upgrading:
 glib2                         x86_64   2.76.6-1.fc38                 updates           2.8 M
 python3                       x86_64   3.11.8-1.fc38                 updates-testing    28 k
 python3-libs                  x86_64   3.11.8-1.fc38                 updates-testing   9.6 M
…
Install  232 Packages
Upgrade    3 Packages
```

Reply:
It's not trivial to add *tested* suggestions here. Added a new one, hopefully, better. Closed this thread as resolved.